sdks/java/managed/src/main/java/org/apache/beam/sdk/managed/ManagedTransformConstants.java

```diff
@@ -50,9 +50,27 @@ public class ManagedTransformConstants {
   private static final Map<String, String> KAFKA_WRITE_MAPPINGS =
       ImmutableMap.<String, String>builder().put("data_format", "format").build();
 
+  private static final Map<String, String> BIGQUERY_READ_MAPPINGS =
+      ImmutableMap.<String, String>builder()
+          .put("table", "table_spec")
+          .put("fields", "selected_fields")
+          .build();
+
+  private static final Map<String, String> BIGQUERY_WRITE_MAPPINGS =
+      ImmutableMap.<String, String>builder()
+          .put("at_least_once", "use_at_least_once_semantics")
```

Comment:
I assume these mappings are the same for all read/write modes ?

Reply:
Yep this essential is mapped to parameters in the schematransform configuration. And in this PR, we are consolidating the file loads and storage api configurations into one